include/RevExt.h

```diff
@@ -34,7 +35,13 @@ struct RevExt {
     : name( name ), feature( feature ), mem( mem ), output( output ) {}
 
   /// RevExt: standard destructor. virtual so that Extensions[i] can be deleted
-  virtual ~RevExt() = default;
+  virtual ~RevExt()                  = default;
+
+  // We do not allow copying, moving or assigning
+  RevExt( const RevExt& )            = delete;
+  RevExt( RevExt&& )                 = delete;
+  RevExt& operator=( const RevExt& ) = delete;
+  RevExt& operator=( RevExt&& )      = delete;
```

Comment:
[Rule of 5](https://en.cppreference.com/w/cpp/language/rule_of_three) to avoid surprises caused by implicit definitions of constructors and assignment operators.